src/components/pages/Offer/StocksManager/StockItem/sub-components/EditAndDeleteControl/EditAndDeleteControl.jsx

```diff
@@ -93,8 +95,14 @@ class EditAndDeleteControl extends PureComponent {
           {!isDeleting && (
             <button
               className="button is-small is-secondary delete-stock"
+              disabled={!isEventDeletable}
               onClick={this.handleOnDeleteClick}
               style={{ width: '100%' }}
+              title={
+                isEventDeletable
+                  ? ''
+                  : 'Les évènements terminés depuis plus de 48h ne peuvent être supprimés'
```

Comment:
```suggestion
                  : 'Les évènements terminés depuis plus de 48 h ne peuvent être supprimés'
```
https://twitter.com/typochef/status/962987516952539136